commons/src/main/java/org/frankframework/util/DateFormatUtils.java

```diff
@@ -48,58 +47,56 @@ public class DateFormatUtils {
 	public static final String FORMAT_FULL_ISO_TIMESTAMP_NO_TZ = "yyyy-MM-dd'T'HH:mm:ss.SSS";
 	public static final String FORMAT_FULL_GENERIC = "yyyy-MM-dd HH:mm:ss.SSS";
 	public static final String FORMAT_DATETIME_GENERIC = "yyyy-MM-dd HH:mm:ss";
-
-	private static final String FORMAT_DATE_ISO = "yyyy-MM-dd";
-	private static final String FORMAT_SHORT_DATE = "dd-MM-yy";
-	private static final String FORMAT_TIME_HMS = "HH:mm:ss";
 	public static final DateTimeFormatter FULL_ISO_FORMATTER = buildFormatter(FORMAT_FULL_ISO);
-
 	public static final DateTimeFormatter FULL_ISO_TIMESTAMP_NO_TZ_FORMATTER = buildFormatter(FORMAT_FULL_ISO_TIMESTAMP_NO_TZ);
 	public static final DateTimeFormatter FULL_GENERIC_FORMATTER = buildFormatter(FORMAT_FULL_GENERIC);
 	public static final DateTimeFormatter GENERIC_DATETIME_FORMATTER = buildFormatter(FORMAT_DATETIME_GENERIC);
+	private static final String FORMAT_DATE_ISO = "yyyy-MM-dd";
 	public static final DateTimeFormatter ISO_DATE_FORMATTER = buildFormatter(FORMAT_DATE_ISO);
+	private static final String FORMAT_SHORT_DATE = "dd-MM-yy";
 	public static final DateTimeFormatter SHORT_DATE_FORMATTER = buildFormatter(FORMAT_SHORT_DATE);
+	private static final String FORMAT_TIME_HMS = "HH:mm:ss";
```

Comment:
Deze regel is een beetje raar..

Reply:
Ik kan het verklaren (private komt _na_ public, tenzij nodig ivm volgorde, zoals hier). Ik snap wat je hier wilt, maar ik kan dit niet afdwingen in de format settings